### ISSUE TITLE
Plots sections: enable linking in the plots tooltips, other improvements

### ISSUE THREAD

**Issue by jorgeorpinel** [iterative/vscode-dvc#1933]
Plots section descriptions.

- [x] put links to DVC docs in those tooltips, probably means they should toggle by a click / ESC, not disappear
- [x] keep all three sections visible (for better education(, if section is empty, show the same message instead of it's content

_Originally posted by @shcheklein in https://github.com/iterative/vscode-dvc/pull/1922#pullrequestreview-1011585467

**Comment by jorgeorpinel** [comment]
> I feel like this repo needs a `ux` issue label.

**Comment by maxagin** [comment]
We had it before. I just want to understand what is the reason for changing as it makes sense if the description is dynamic. Please clarify it for me. Thanks

<img width="842" alt="Screen Shot 2022-06-21 at 4 22 44 PM" src="https://user-images.githubusercontent.com/98249521/174891003-6495d0cd-2fff-495c-9d6b-58e917706d01.png">

**Comment by shcheklein** [comment]
> I think @shcheklein meant that instead of tooltips we should have the text visible under each Plot type header

No, we don't want this text to be permanently visible. Only as (potential) way for placeholders when there are not plots in the section.

tooltip should stay as a tooltip but react properly considering that there will be links

**Comment by shcheklein** [comment]
I've updated the description, there was a bug -  `should not toggle` -> ` should toggle`

**Comment by maxagin** [comment]
I think I have a good solution for this. I will share the concept soon

**Comment by maxagin** [comment]
@jorgeorpinel , here is a solution I believe in :) Sharing the example below. Tomorrow I will design the example of a hint including a button (or another link example).

Concept: show hint on hover with the mouse over the transparent area so the user can move the cursor into the hint area without it disappearing. When the mouse is out of the hint area - we hide it.

This will work also for other similar sections, such as coming errors: DVC errors that may occur during or after running experiment #1636

Example from our favourite GitHub

https://user-images.githubusercontent.com/98249521/175198240-0b9379ca-967d-46e6-83d0-b645e4959874.mov

**Comment by maxagin** [comment]
Minor: the hints are not context menus, this is why I suggest we will have them centred in relation to the hovered element. Thanks

**Comment by maxagin** [comment]
The [Figma spec](https://www.figma.com/file/O9foIQafZV2eEiMlextVXa/1635-%E2%80%A2-Meaning-of-each-section-in-Plots-view?node-id=1294%3A4169) was updated with the new concept. Please let me know if you have any questions or comments.

**Comment by maxagin** [comment]
This is a transparent area that appears when the icon has hovered. Find more information in the [Figma spec](https://www.figma.com/file/O9foIQafZV2eEiMlextVXa/1635-%E2%80%A2-Meaning-of-each-section-in-Plots-view?node-id=1294%3A4169)

We could use a button here, but I think that text link works better for this case.

<img width="589" alt="Screen Shot 2022-06-23 at 2 42 30 PM" src="https://user-images.githubusercontent.com/98249521/175374340-a06157e3-2089-4f49-8f5f-6864672ea202.png">

**Comment by mattseddon** [comment]
Relates to #1954.

**Comment by maxagin** [comment]
Hey. What is the status of this ticket? Are we all agreed it is ready for development? Thanks

**Comment by sroy3** [comment]
What are the links we want to put it? Once #2140 is merged, everything is ready to put in links, but I can also add them inside that PR.

**Comment by shcheklein** [comment]
@sroy3 let's wait for https://github.com/iterative/dvc.org/pull/3830 to be merged, @dberenbaum has changed the structure there. We can put those link later, we can create and assign to me a small followup ticket so that we don't block this PR.
